src/bucket/BucketManager.h

```diff
@@ -211,5 +214,19 @@ class BucketManager : NonMovableOrCopyable
     virtual void shutdown() = 0;
 
     virtual bool isShutdown() const = 0;
+
+    // Load the complete state of the ledger from the provided HAS. Throws if
```

Comment:
maybe a little warning that this will use a lot of memory?

Reply:
As a comment? Ok, I'll make a note.